src/test/java/ir/sahab/dockercomposer/DockerComposeRunnerTest.java

```diff
@@ -3,18 +3,46 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 import org.assertj.core.api.Condition;
+import org.junit.Assert;
+import org.junit.ClassRule;
 import org.junit.Test;
 import org.zeroturnaround.exec.ProcessExecutor;
 
+import java.io.IOException;
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.util.Map;
+import java.util.concurrent.TimeoutException;
 
 public class DockerComposeRunnerTest {
 
     private static final int PORT = 2181;
     private static final Condition<Service> accessible = new ServiceAccessible(PORT);
 
+
+    @ClassRule
+    public static DockerCompose dockerCompose = DockerCompose.builder()
```

Comment:
I think here it's better to define service in the method instead of class rule. It's not used in other tests.